calico_node/Makefile

```diff
@@ -63,6 +34,17 @@ SYSTEMTEST_CONTAINER?=calico/test:$(SYSTEMTEST_CONTAINER_VER)
 # Ensure that the dist directory is always created
 MAKE_SURE_BIN_EXIST := $(shell mkdir -p dist)
 
+.PHONY: print-version
+print-version:
+	$(info $$RELEASE_STREAM is ${RELEASE_STREAM})
+    $(info $$BIRD_VER is ${BIRD_VER})
```

Comment:
these info statements probably need a <tab> indent, not sure how gmake interprets actions with space indents.